routes/api/v1/foods.js

```diff
@@ -16,7 +16,14 @@ router.get('/', function(req, res, next) {
       res.status(500).json({ error });
     });
 });
-
+// const Food = require('../../../models/food')
```

Comment:
Why did you comment this out? Was it breaking things with it?

Reply:
currently, it is not necessary. I think if we switch the foods routes to use Objection, then we may need it.